third_party/CMakeLists.txt

```diff
@@ -38,6 +38,37 @@ if(DD_APPSEC_BUILD_EXTENSION)
     else()
         target_compile_definitions(mpack PUBLIC MPACK_STDIO=0 MPACK_WRITE_TRACKING=1 MPACK_BUFFER_SIZE=131072)
     endif()
+
+    # DDTrace ZAI
+    find_program(PHP_CONFIG php-config)
+    execute_process(
+            COMMAND ${PHP_CONFIG} --includes
+            RESULT_VARIABLE PHP_CONFIG_INCLUDES_RESULT
+            OUTPUT_VARIABLE PHP_CONFIG_INCLUDES
+            ERROR_VARIABLE PHP_CONFIG_INCLUDES_ERR
+            OUTPUT_STRIP_TRAILING_WHITESPACE
+    )
+
+    set(ZAI_INCLUDES "${CMAKE_CURRENT_SOURCE_DIR}/ddtrace/zend_abstract_interface")
+    if("${PHP_CONFIG_INCLUDES_RESULT}" STREQUAL "0")
+        string(REPLACE "-I " "" PHP_INCLUDES "${PHP_CONFIG_INCLUDES}")
+        string(REPLACE "-I" "" PHP_INCLUDES "${PHP_CONFIG_INCLUDES}")
+        string(REPLACE " " ";" PHP_INCLUDES "${PHP_INCLUDES}")
```

Comment:
not using `-isystem`, which I think is what changed here, can result in warnings coming from PHP code